package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.124.0",
+  "version": "2.125.0",
```

Comment:
So... here's the thing. This PR does include breaking changes for MessagingAttachment (adding new required props). The very next thing I'm going to do is to go into LP and add those required props to every instance of MessagingAttachment, but they're still breaking. Because of this, the PR template suggests `npm version major`, but I didn't do that... Should I do that?

Reply:
MessagingAttachment is a beta component (as I believe all Messaging components are), so we're allowed to make breaking changes with only a minor version bump! 🎉